Makefile

```diff
@@ -32,8 +32,8 @@ start_dev: ## start with dev build image, with reviewer-mocks mocking continuum
 	${DOCKER_COMPOSE} pull reviewer-mocks nginx
 	$(MAKE) build_dev
 	${DOCKER_COMPOSE} up -d reviewer-client reviewer-mocks
-	./.scripts/docker/wait-healthy.sh reviewer-client_app 210
```

Comment:
Do we not need to wait for them to be up anymore?

Reply:
accidental commit